draft-packages/pagination/package.json

```
@@ -0,0 +1,44 @@
+{
+  "name": "@kaizen/draft-pagination",
+  "version": "1.0.0",
+  "description": "The draft Pagination component",
+  "scripts": {
+    "prepublish": "tsc --project tsconfig.dist.json",
+    "build": "yarn clean && yarn prepublish",
+    "build:watch": "yarn clean && yarn prepublish --watch",
+    "clean": "rimraf '**/*.d.ts' '**/*.js' '**/*.map'"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cultureamp/kaizen-design-system.git",
+    "directory": "draft-packages/pagination"
+  },
+  "bugs": {
+    "url": "https://github.com/cultureamp/kaizen-design-system/issues"
+  },
+  "files": [
+    "**/*",
+    "!**/*.ts",
+    "!**/*.tsx",
+    "**/*.d.ts",
+    "!**/*.spec.*",
+    "!**/*.snap",
+    "!draft-templates",
+    "!stories",
+    "!docs",
+    "!tsconfig.dist.json"
+  ],
+  "author": "",
+  "private": false,
+  "license": "MIT",
+  "dependencies": {
+    "@kaizen/component-library": "^10.0.3",
+    "classnames": "^2.3.1",
+    "path-to-regexp": "^6.2.0"
+  },
+  "devDependencies": {},
+  "peerDependencies": {
+    "@kaizen/design-tokens": "^2.5.0",
```

I think the current version of design-tokens is 2.9.3. If theres no reason not to bump this, i think we should.
```suggestion
    "@kaizen/design-tokens": "^2.9.3",
```

---

I realised this was a peer sorry so probably doesn't matter!